test/testHelpers.ts

```diff
@@ -26,10 +28,16 @@ export interface IAssignmentTestCase {
   expectedAssignments: IValue[];
 }
 
-export function readMockRacResponse(): Record<string, IExperimentConfiguration> {
+export function readMockRacResponse(): { flags: Record<string, IExperimentConfiguration> } {
   return JSON.parse(fs.readFileSync(TEST_DATA_DIR + MOCK_RAC_RESPONSE_FILE, 'utf-8'));
 }
 
+export function readMockObfuscatedRacResponse(): {
```

Comment:
I'm seeing I could combine this and `readMockRacResponse` by making the filename an argument ..